### PR TITLE
KES User Interface

### PR DESCRIPTION
**Pull request by reivaj05**
**To Test this PR**

1. [Run KES Server](https://github.com/minio/kes/wiki/MinIO-Object-Storage#kes-server-setup)
2. `make assets` 
3. `make kes`
4. run `./kes ui`

### CODE REVIEW

**Review by cniackz — APPROVED** [review]
LGTM, but don't merge until Andreas approve it.

File: web-app/src/screens/console/common/FormLayout.tsx
Comment (bexsoft) [inline web-app/src/screens/console/common/FormLayout.tsx:1]: We can use FormLayout from mds here

File: web-app/src/screens/console/common/InputBoxWrapper.tsx
Comment (bexsoft) [inline web-app/src/screens/console/common/InputBoxWrapper.tsx:1]: We can use InputBox from mds here

File: web-app/src/screens/console/common/PageLayout.tsx
Comment (bexsoft) [inline web-app/src/screens/console/common/PageLayout.tsx:1]: We can use PageLayout from mds here

File: web-app/src/screens/console/common/SectionTitle.tsx
Comment (bexsoft) [inline web-app/src/screens/console/common/SectionTitle.tsx:1]: We can use SectionTitle from mds here

File: web-app/src/screens/console/encryption/api/StatusIcon.tsx
Comment (bexsoft) [inline web-app/src/screens/console/encryption/api/StatusIcon.tsx:1]: Can this be included in mds?

File: web-app/src/screens/console/encryption/keys/KeysIcon.tsx
Comment (bexsoft) [inline web-app/src/screens/console/encryption/keys/KeysIcon.tsx:1]: Can this be included in mds?

File: web-app/package.json
Comment (bexsoft) [inline web-app/package.json:29]: Please update mds to 0.3.2

**Review by bexsoft — DISMISSED** [review]
Can we send all icons to mds library, please?

**Comment by allanrogerr** [comment]
@reivaj05 Im facing this when building. Any tips?
```
➜  kes git:(kes-ui-wip) ✗ make kes
Building kes binary to './kes'
# github.com/minio/kes/internal/api
../../internal/api/api.go:74:22: undefined: http.NewResponseController
../../internal/api/router.go:105:8: undefined: http.NewResponseController
../../internal/api/router.go:148:8: undefined: http.NewResponseController
make: *** [kes] Error 2
```

**Comment by allanrogerr** [comment]
@reivaj05 Is this still required?

**Comment by harshavardhana** [comment]
I am closing this PR as stale and outdated.

**Pull request closed without merging**
